targets/mega65/vic4.c

```diff
@@ -1065,6 +1063,22 @@ static void vic4_render_fullcolor_char_row ( const Uint8* char_row, int glyph_wi
 }
 
 
```

Comment:
Sorry for the question again, just make it clear :) So the part BELOW this comment looks like what I expected. What about the changes above? Are those really intended? I can't see it's connected to the implementation of the nybl mode video.

Reply:
Looking at the file history, that comments were suited for better debugging the NTSC/PAL mode change.

Reply:
Yes but the !in_hypervisor again changed position what I worry for, but if you say it's intended that way, of course I believe you ;)

Reply:
I think it's the correct position -- may be the performance regression was due to this? e.g:  recalculating modes on every frame access.   The regressed output should have a lot of VIC4: border top=xxx etc.